cmd/src/args/permissions/global.rs

```diff
@@ -0,0 +1,395 @@
+use iggy::models::permissions::GlobalPermissions;
+use std::str::FromStr;
+
+#[derive(Debug, PartialEq)]
+pub(super) enum GlobalPermission {
+    ManageServers,
+    ReadServers,
+    ManageUsers,
+    ReadUsers,
+    ManageStreams,
+    ReadStreams,
+    ManageTopics,
+    ReadTopics,
+    PollMessages,
+    SendMessages,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub(crate) struct GlobalPermissionError(String);
+
+impl FromStr for GlobalPermission {
+    type Err = GlobalPermissionError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "manage_servers" => Ok(GlobalPermission::ManageServers),
```

Comment:
Maybe, we could merge the lines into following expression `"m_srv" | "manage_servers"`?

Reply:
Done